### PR TITLE
Fix when throwing an exception in a parallel callback

### PR DESCRIPTION
**Pull request by joelwurtz**

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -56,6 +87,10 @@ function parallel(callable ...$callbacks): array
         }
     }
 
+    if ($exceptions) {
+        throw new \RuntimeException('One or more exceptions were thrown in parallel');
```
Comment (pyrech) [inline src/functions.php:91]: Should we throw a custom exceptions allowing to retrieve all inner exceptions?

Comment (joelwurtz) [reply src/functions.php:91]: Let's see if someone has an use case for that, don't want to add too much here, it's mainly to fix a bug

Comment (lyrixx) [inline src/functions.php:91]:
```suggestion
        throw new \RuntimeException('One or more exceptions were thrown in parallel.');
```

**Review by lyrixx — COMMENTED** [review]
On little comment, otherwise 👍🏼

**Pull request merged**